dask_sql/physical/rel/custom/create_model.py

```diff
@@ -45,16 +42,12 @@ class CreateModelPlugin(BaseRelPlugin):
       want to set this parameter.
     * wrap_predict: Boolean flag, whether to wrap the selected
       model with a :class:`dask_sql.physical.rel.custom.wrappers.ParallelPostFit`.
-      Have a look into the
-      [dask-ml docu](https://ml.dask.org/meta-estimators.html#parallel-prediction-and-transformation)
-      to learn more about it. Defaults to false. Typically you set
-      it to true for sklearn models if predicting on big data.
+      Defaults to false. Typically you set it to true for
+      sklearn models if predicting on big data.
```

Comment:
Unrelated to this PR , can you file an issue to clean up the `wrap_predict` and `wrap_fit` arguments. I think we can get rid of this or do a better default based on the class name of the model. 

For `sklearn` and `single gpu ` cuML models, switch this to true else switch this to False.